docs/source/install.rst

```diff
@@ -0,0 +1,77 @@
+************
+Installation
+************
+
+Standard Package Requirements
+#############################
+
+qgrad depends on the following standard packages
+
+.. cssclass:: table-striped
+
++----------------+--------------+-----------------------------------------------------+
+| Package        | Version      | Details                                             |
++================+==============+=====================================================+
+| **Python**     | 3.7+         | Not tested for lower versions.                      |
++----------------+--------------+-----------------------------------------------------+
+| **NumPy**      | 1.19+        | Not tested on lower versions.                       |
++----------------+--------------+-----------------------------------------------------+
+| **SciPy**      | 1.3+         | Not tested for lower versions.                      |
++----------------+--------------+-----------------------------------------------------+
+| **Matplotlib** | 3.2.2+       | Needed for tutorials.                               |
++----------------+--------------+-----------------------------------------------------+
+| **Cython**     | 0.21+        | Needed for working with QuTiP.                      |  
++----------------+--------------+-----------------------------------------------------+
+| **pytest**     | 5.1.3+       | For running tests                                   |
++----------------+--------------+-----------------------------------------------------+
+
+
+In addition to the standard standard packages listed above,
+`JAX <https://github.com/google/jax>`_ is needed to work with qgrad. JAX can be 
+installed with the following commands::
+
```

Comment:
```suggestion

.. code-block:: bash

```

Reply:
yeah, :: works for bash too. I am wondering then including .. code- block:: bash wouldn't be needed?